R/add_transition_risk_NA_share.R

```diff
@@ -0,0 +1,98 @@
+add_transition_risk_NA_share <- function(data) {
+  product <- data |>
+    unnest_product() |>
+    add_transition_risk_NA_share_at_product_level()
+
+  company <- data |>
+    unnest_company() |>
+    select_and_join_transition_risk_NA_share_at_company_level(product)
+
+  tilt_profile(nest_levels(product, company))
+}
+
+add_transition_risk_NA_share_at_product_level <- function(data) {
+  data |>
+    fill_benchmark_tr_score() |>
+    compute_transition_risk_NA_amount_all() |>
```

Comment:
Consider removing "compute_" since it adds little value.

ChatGTP
> Hadley Wickham discusses naming conventions for functions in the tidyverse, stating that function names should generally be verbs, unless there is a common verb used by many functions (like "compute"). In that case, it is better to avoid repeating the common verb and instead focus on the noun to make the function names more concise and readable. For example, instead of naming functions compute_mean() and compute_median(), it would be more concise to name them mean() and median().